datanode/gateway/graphql/resolvers.go

```diff
@@ -855,6 +855,161 @@ func (r *myQueryResolver) EstimateOrder(
 	}, nil
 }
 
+func (r *myQueryResolver) EstimateFees(
```

Comment:
Didn't we already have an EstimateFees API?

Reply:
No, it was part of `EstimateOrder`, but that uses `EstimateMargin` which I've marked as deprecated (would like us to delete it once frontend switches to the new method), so I've extracted the fees bit to a separate method so they can use that one.

Reply:
hum alright.